product-service/functions/createProduct.ts

```diff
@@ -0,0 +1,50 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import { productRepository } from '../repository';
+import {Product} from "../types";
+
+const baseResponse = {
+    headers: {
+        'Access-Control-Allow-Origin': '*',
+    }
+}
+
+function validate({title, description, price, author, count}: Omit<Product, 'id'>) {
```

Comment:
Please, move to separate module

Reply:
Thank you for review!
About "manage products". As I understand, it's not part of current task because it is optional, must have, but optional. I'm going fix it in feature.
![image](https://user-images.githubusercontent.com/7376989/98698430-72e77900-238f-11eb-9762-fac0e383d6de.png)

With third-party validator library I agree with you. Will be added in feature too =)